schemas/groups/performance.json

```diff
@@ -64,6 +64,18 @@
       "$ref": "../definitions.json#/definitions/numberValue",
       "units": "m/s"
     },
+    
+    "targetAngle": {
+      "description": "The true wind gybe or beat angle for the best velocity made good downwind based on current polar and trueWindSpeed.",
```

Comment:
A bit inconsistent, should probably say down/upwind and not just downwind.

And isn't current polar really a function of trueWindSpeed, should the polar mention be omitted?